scripts/cibuild/BuildValidator.ps1

```diff
@@ -19,38 +19,82 @@ param
     [string]$BuildOutputTargetPath,
     [Parameter(Mandatory=$True)]
     [string]$BuildRTM,
-    [switch]$ValidateVsix
+    [Parameter(Mandatory=$True)]
+    [string]$BuildConfiguration,
+    [switch]$ValidateVsix,
+    [switch]$ValidateSigning
 )
 
 
 if ($BuildRTM -eq 'false')
 {   
 
+    $result = 0
     $NuGetClientRoot = $env:BUILD_REPOSITORY_LOCALPATH
-    $NuGetValidator = [System.IO.Path]::Combine($NuGetClientRoot, 'packages', 'NuGetValidator.1.3.7', 'tools', 'NuGetValidator.exe')
-    $VsixLogOutputDir = [System.IO.Path]::Combine($BuildOutputTargetPath, 'LocalizationValidation', 'vsix' )
-    $LocalizationRepository = [System.IO.Path]::Combine($NuGetClientRoot, 'submodules', 'NuGet.Build.Localization', 'localize', 'comments', '15')
+    $NuGetValidator = [System.IO.Path]::Combine($NuGetClientRoot, 'temp', 'NuGetValidator.1.4.0.3', 'tools', 'NuGetValidator.exe')
```

Comment:
Before calling NuGetValidator.exe or sn.exe you should verify their Authenticode signature using [Get-AuthenticodeSignature](https://technet.microsoft.com/en-us/library/ee176840.aspx) and fail immediately if the signature is not valid.